README.md

```diff
@@ -354,6 +362,74 @@ func Write(ctx context.Context, db sqx.Queryable) interface {
 }
 ```
 ----
+### Why `sqx`?
```

Comment:
I think you should call back to how our `sqx.Read[User]` would look as a point of comparison. Maybe it's just the PR view, but it seemed like `GetUsers()` was defined pretty far above. Even just a link to "recall how our GetUsers function was written..." would help